Source/Bugsnag.m

```diff
@@ -51,9 +51,13 @@ + (void)startBugsnagWithApiKey:(NSString *)apiKey {
 
 + (void)startBugsnagWithConfiguration:(BugsnagConfiguration *)configuration {
     @synchronized(self) {
-        bsg_g_bugsnag_notifier =
-        [[BugsnagNotifier alloc] initWithConfiguration:configuration];
-        [bsg_g_bugsnag_notifier start];
+        if (configuration != nil && configuration.apiKey != nil) {
```

Comment:
Might be nicer to have a configuration based validation method. `[configuration isValid]` for example, rather than the Bugsnag class having to know what that means.